examples/knowledge_tuning/instructlab/document_pre_processing.py

```
@@ -0,0 +1,75 @@
+# %% [markdown]
+# ### Using docling v2
+
+# %%
+data_dir = 'document_collection/ibm-annual-report'
+# !OMP_NUM_THREADS=32 mamba run -n docling python ../scripts/docparser_v2.py --input-dir {data_dir} --output-dir {data_dir} --c docling_v2_config.yaml
+
+# %% [markdown]
+# ### Create Seed Examples
+
+# %%
+from utils import DocProcessor
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)